model/user/user.go

```diff
@@ -63,6 +63,7 @@ type UserSettings struct {
 	SlackUsername    string                  `bson:"slack_username,omitempty" json:"slack_username,omitempty"`
 	Notifications    NotificationPreferences `bson:"notifications,omitempty" json:"notifications,omitempty"`
 	UseSpruceOptions UseSpruceOptions        `json:"use_spruce_options" bson:"use_spruce_options"`
+	DateFormat       string                  `json:"date_format" bson:"date_format"`
```

Comment:
So its not exactly a free form string the available options are defined [here](https://github.com/evergreen-ci/spruce/pull/1413/files#diff-3910226f5c98a3ff714bd3ab3ae991625e4eb57b7446a7f52ad6531f5a938ddbR177) and are selectable from a dropdown.
![image](https://user-images.githubusercontent.com/4605522/186165417-608863df-7db3-4f79-a483-f7fb7a7ac942.png)

Since this field is only used on the UI i don't think it makes sense to maintain a second list to validate the options on the backend.

Reply:
perfect. I agree.